src/components/panel/demoGroups/style.global.css

```diff
@@ -0,0 +1,80 @@
+.menu-panel-container {
+  pointer-events: all;
```

Comment:
@ErinCoughlan When specifying `propagateContainerEvents` in the panel config, the panel CSS applies `pointer-events: none` to the entire panel DOM element. This was added to the `menu-panel-container` class and then the class was added with the `panelClass` property of the panel config, in order to allow for the user to interact with the menu.